### PR TITLE
Davids sida

### PR DESCRIPTION
**Pull request by iths-david-larsson**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,11 +43,30 @@ <h1>This we're not that good at</h1>
...
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </article>
             <article class="Moa">Moas sektion</article>
             <article class="Daniel">Daniels sektion</article>
             <article class="Alex">Alex sektion</article>
         </div>
+        <script
```
Comment (iths-daniel-holbling) [inline index.html:65]: Why jQuery?!

**Review by iths-daniel-holbling — COMMENTED** [review]
too much jQuery, but ok

File: index.html [hunk 2]
```diff
@@ -43,11 +43,30 @@ <h1>This we're not that good at</h1>
             </section>
         </div>
         <div class="personal-wrapper">
-            <article class="David">Davids sektion</article>
+            <article class="David">
```
Comment (moa-granberg) [inline index.html:46]: classes lowercase

**Review by moa-granberg — APPROVED** [review]
Cool! Also mobile 👍

**Review by iths-alexander-rusiecki — APPROVED** [review]

**Pull request merged**